app/controllers/collection_items_controller.rb

```diff
@@ -26,28 +27,30 @@ def index
     if @collection && @collection.user_is_maintainer?(current_user)
       @collection_items = @collection.collection_items.include_for_works
       @collection_items = case
-      when params[:approved]
-        @collection_items.approved_by_collection
-      when params[:rejected]
-        @collection_items.rejected_by_collection
-      else
-        @collection_items.unreviewed_by_collection
-      end
+                          when params[:approved]
+                            @collection_items.approved_by_collection
+                          when params[:rejected]
+                            @collection_items.rejected_by_collection
+                          when params[:invited]
+                            @collection_items.invited_by_collection
+                          else
+                            @collection_items.unreviewed_by_collection
+                          end
     elsif params[:user_id] && (@user = User.find_by_login(params[:user_id])) && @user == current_user
       @collection_items = CollectionItem.for_user(@user).includes(:collection)
       @collection_items = case
-      when params[:approved]
-        @collection_items.approved_by_user
-      when params[:rejected]
-        @collection_items.rejected_by_user
-      else
-        @collection_items.unreviewed_by_user
-      end
+                          when params[:approved]
+                            @collection_items.approved_by_user.approved_by_collection
+                          when params[:rejected]
+                            @collection_items.rejected_by_user
+                          else
+                            @collection_items.unreviewed_by_user
+                          end
     else
       flash[:error] = ts("You don't have permission to see that, sorry!")
       redirect_to collections_path and return
     end
-    
+
     sort = "created_at DESC"
     # case params[:sort]
```

Comment:
This large block of code has been commented out since 2011. I think we can safely assume we don't need it. But not a blocker, as we can also safely assume we can live with it a bit longer!